EZP-28161: Enable php-cs-fixer aligned with v2.7.1 settings

alongosz opened the pull request:
# Fixes [EZP-28161](https://jira.ez.no/browse/EZP-28161)

This PR introduces `.php_cs` config and aligns CS.
To simplify fixing code it adds dev dependency on `php-cs-fixer` and introduces composer command:
```
composer fix-cs
```
which runs fixer on all files.

**TODO**:
- [x] Create php-cs-fixer config (`.php_cs`)
- [x] Add `php-cs-fixer` dev dependency to composer pointing to `~2.7.1`
- [x] Add composer command `fix-cs`
- [x] Fix CS
- [x] Travis: drop PHP 5.5 and add PHP 7.1 testing
- [x] Enable CS check on Travis (with disabled XDebug)

andrerom commented on SystemInfo/Collector/DoctrineDatabaseSystemInfoCollector.php:
```diff
@@ -33,7 +33,7 @@ public function __construct(Connection $db)
      *  - name
      *  - host
-     *  - username
+     *  - username.
```
This one is strange, but ok 🙃

alongosz replied:
Apparently lists are treated as a sentence that requires dot at the end. From spelling POV (at least in Polish) it makes perfect sense ;)

alongosz commented:
Changes since last review:
- Dropped PHP 5.5 testing
- Created test matrix and enabled PHP 7.1 testing
- When testing PHP 7.1 added test by php-cs-fixer
- Removed XDebug when running php-cs-fixer per its suggestion (initially I tried to remove it globally, but coverage report by phpunit needs it).

This one is ready.
